locale/ca-es/search_wolfie.intent

```diff
@@ -0,0 +1,10 @@
+(cerca|busca|consulta|demana) a (volfram alfa|wolfram alpha) {query}
+(cerca|busca|consulta|demana) {query} a (volfram alfa|wolfram alpha)
+(cercar|buscar|consultar|demanar) a (volfram alfa|wolfram alpha) {query}
+(cercar|buscar|consultar|demanar) {query} a (volfram alfa|wolfram alpha)
+què diu (volfram alfa|wolfram alpha) (sobre|de) query}
```

Comment:
seems like you missed a { in a few of those `query}` should be `{query}`

Reply:
Fixed!